src/resources/batches/BatchForm.tsx

```diff
@@ -115,7 +117,13 @@ const BatchForm = (props: FormProps): React.ReactElement => {
       <EditToolBar
         type='button'
         mutationOptions={{
-          onSuccess: ({ id }: { id: number }) => { setItemId(id) }
+          onSuccess: ({ id }: { id: number }) => {
+            setItemId(id)
+            if (!isEdit) {
+              const path = `/${constants.R_BATCHES}/${id}/show`
+              redirect(path)
+            }
+          }
```

Comment:
But @TahaKhanAbdalli - isn't this a more robust way of going to the `show` page?
![image](https://github.com/DeepBlueCLtd/RCO/assets/1108513/e02edf39-8f26-4e84-a0de-fa7e92eb114f)

Reply:
I have tried but as the `toolbar` is using `mutationOptions`, it does not trigger `redirect` on `Create`